grpc/execution/server.go

```diff
@@ -92,6 +92,13 @@ func (s *ExecutionServiceServer) DoBlock(ctx context.Context, req *executionv1.D
 		return nil, fmt.Errorf("failed to insert block into blockchain (n=%d)", n)
 	}
 
+	// remove txs from original mempool
+	for _, block := range blocks {
+		for _, tx := range block.Transactions() {
+			s.eth.TxPool().RemoveTx(tx.Hash())
+		}
+	}
```

Comment:
```suggestion
		for _, tx := range block {
			s.eth.TxPool().RemoveTx(tx.Hash())
		}
```

can be simplified as there's only one block